FeatureTest.md

```diff
@@ -1,3 +1,18 @@
+# Motivating Scenarios
+1. In MVP + 1 (the version of the WebAssembly after MVP), [`i32.min_s`](FutureFeatures.md#additional-integer-operations) is introduced. A developer updates her toolkit and targets the new features. The compiler leverages `i32.min_s` when compiling a module the developer wrote. The developer's WebAssembly module works correctly both on execution environments at MVP, as well as those updated on MVP + 1.
```

Comment:
I think we've tried going gender-neutral everywhere, so "developers update their".